### DISCUSSION TITLE
Audio via USB and image customization

### DISCUSSION THREAD

**Discussion by mtc86** [opencardev/crankshaft#627] [Q&A]
Good morning everyone,
I have two questions:
The first is that when I make a call with my cellphone connected via USB, the audio is not transmitted by AndroidAuto but goes to my cellphone.
The second is how do I change the images on the main screen, tried to replace the images in /boot/crankshaft/ but without success, the originals remain.
Thank you for your cooperation and have a good day.

**Comment by Jirradia** [comment]
Does your setup have a microphone?

For your second question did you check if your image is in the same format as the original image? I think they're PNG. just make sure that have the same file name and you should be fine. Also, the images don't scale you should use the same resolution as the original image.
